pkg/plugin/v3/scaffolds/internal/templates/config/controller/controller_suitetest.go

```diff
@@ -38,7 +38,7 @@ type SuiteTest struct {
 // SetTemplateDefaults implements file.Template
 func (f *SuiteTest) SetTemplateDefaults() error {
 	if f.Path == "" {
-		if f.MultiGroup {
+		if f.MultiGroup && f.Resource.Group != "" {
 			f.Path = filepath.Join("controllers", "%[group]", "suite_test.go")
 		} else {
 			f.Path = filepath.Join("controllers", "suite_test.go")
```

Comment:
We need to merge the fix #1682 and the will be required add more a conditional here. https://github.com/kubernetes-sigs/kubebuilder/pull/1682/files#diff-71066f95caa43267d8f2443a3c222fbeR57-R63  

If is multigroup and the resource is not empty then the path of the CRDs is not ../../ otherwise it is ../

Reply:
Got it, I will wait for this PR to be merged.

Reply:
Completed the last nit :)